packages/types/src/utility-types.ts

```diff
@@ -8,3 +10,9 @@ export type Constructor<T = unknown> = new (...args: any[]) => T;
 export type Awaitable<T> = T | PromiseLike<T>;
 
 export type AllowedHttpMethods = 'GET' | 'HEAD' | 'POST' | 'PUT' | 'DELETE' | 'TRACE' | 'OPTIONS' | 'CONNECT' | 'PATCH';
+
+export type RouterRoutes<Context, UserData extends Dictionary> = {
```

Comment:
don't have strong options about this type name, feel free to suggest better

Reply:
Good point, moving...
GetUserDataFromRequest is already defined there
![image](https://user-images.githubusercontent.com/697301/235305859-7776145b-6430-495f-93f1-24a6d6693e09.png)